checkout/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from tickets.models import Ticket
+
+# Create your models here.
+class Order(models.Model):
+    full_name = models.CharField(max_length=50, blank=False)
```

Comment:
Order should be connected to the User object as a Foreign Key so that it is possible to find all orders belonging to the User